### PR TITLE
Registering saifeemustafaq

### PR DESCRIPTION
**Pull request by saifeemustafaq**
Thankyou Maintainers!

You are doing an awesome job. Please accept my pull request!

### CODE REVIEW

**Comment by gkanishk** [comment]
Hey @saifeemustafaq ,
Past projects won't be considered for the challenge only projects done after mid-march 2020 are considered for the challenge, so please update accordingly.

File: content/participants/ttjc@saifeemustafaq/index.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+name: "Mustafa Saifee"
+portfolio: "https://www.saifeemustafa.com/"
+github-link: "https://github.com/saifeemustafaq/"
+linkedin-link: "https://www.linkedin.com/in/saifeemustafaq/"
+looking-for: "internship"
+description: "Microsoft Azure | Python | DevOps"
+
+------
+
+> Hi! I am a Mustafa Saifee, let me walk you through my intro, projects and blogs.
```
Comment (gkanishk) [inline content/participants/ttjc@saifeemustafaq/index.md:13]: minor typo remove 'a'

Comment (gkanishk) [reply content/participants/ttjc@saifeemustafaq/index.md:13]: Hi I am a Mustafa

- Are you any non living thing or a human?? What do you mean "a" Mustafa

Comment (khusharth) [reply content/participants/ttjc@saifeemustafaq/index.md:13]: Instead of I am **a** Mustafa --->  it should be I am Mustafa

Comment (saifeemustafaq) [reply content/participants/ttjc@saifeemustafaq/index.md:13]: Haha. I am so sorry. I completely missed it. Thankyou for pointing that and sorry for that silly mistake

Comment (saifeemustafaq) [reply content/participants/ttjc@saifeemustafaq/index.md]: @gkanishk I have done the suggested changes. Thank you

**Comment by gkanishk** [comment]
Hey @saifeemustafaq ,
Welcome on-board it's good to see a Gold Student partner on-board, happy to see you!!

**Comment by saifeemustafaq** [comment]
Thank you for the welcome @gkanishk :) 
Looking forward to working with you and the team.
Can you point me out to the next steps please?

**Comment by gkanishk** [comment]
> Thank you for the welcome @gkanishk :) 
> Looking forward to working with you and the team.
> Can you point me out to the next steps please? 

Sure find the challenge details here
https://2020.teamtanay.jobchallenge.dev/challenge-details/
Also check the faqs section.

**Pull request merged**